dev_ops/sql/Tables/CreateContracts.sql

```diff
@@ -14,6 +14,5 @@ CREATE TABLE contracts (
 	CONSTRAINT valid_timestamps CHECK (
 	    proposed_at < accepted_at AND -- the contract must be accepted after its proposed
 		accepted_at < expires_at AND -- the contract must be accepted before it is expired
-		completed_at <= expires_at AND -- the contract must be completed before it is expired
```

Comment:
Was this causing problems when its null?

Reply:
I had to remove this for the case when a contract is expired but its not completed until  the SQS logic runs. In this case completed_at will be after expires_at

Reply:
If the contract is expired, why is completed being set at all? I'm a bit confused about this.

Reply:
Ill talk to you offline for this